subprojects/snapshots/src/main/java/org/gradle/internal/snapshot/PathUtil.java

```diff
@@ -167,13 +172,13 @@ static boolean equalChars(char char1, char char2, CaseSensitivity caseSensitivit
      *
      * The size of the common prefix does not include the last line separator.
      */
-    public static int sizeOfCommonPrefix(String path1, String path2, int offset, CaseSensitivity caseSensitivity) {
+    public static int sizeOfCommonPrefix(String relativePath, String absolutePath, int offset, CaseSensitivity caseSensitivity) {
```

Comment:
I think calling this `lengthOfCommonPrefix` would clarify things. We usually talk about length not size when it comes to strings.

Reply:
Renamed.